### DISCUSSION TITLE
How Data Centric is Uno?

### DISCUSSION THREAD

**Discussion by karimkhoja13** [unoplatform/uno#11427] [Q&A]
How does Uno handle data?

What is the system to manage databases using Uno?

**Comment by jeromelaban** [comment]
Thanks for the question! Uno does not facilitate databases management in the same way WinUI does not. You can use any database that you the platform you're running on already provides support for.

That being said, for WebAssembly specifically, we're providing a compatible build of SQLite. You can find a sample of [EFCore with SQLite on multiple platforms](https://github.com/unoplatform/Uno.Samples/tree/master/UI/EFCoreSQLiteSample).
